packages/commons/package.json

```diff
@@ -31,15 +32,19 @@
     "pagopa-interop-models": "workspace:*",
     "pg-promise": "^11.5.0",
     "ts-pattern": "^5.0.6",
+    "uuid": "^9.0.0",
     "winston": "^3.9.0",
     "zod": "^3.21.4"
   },
   "devDependencies": {
+    "@anatine/zod-mock": "^3.13.1",
+    "@testcontainers/postgresql": "^10.2.2",
     "@types/express": "^4.17.17",
     "@types/jsonwebtoken": "^9.0.2",
     "@types/node": "^20.3.1",
     "eslint": "^8.44.0",
     "prettier": "^2.8.8",
+    "testcontainers": "^10.2.2",
```

Comment:
Could we remove `@testcontainers/postgresql` and `testcontainers` dependencies from commons? I think it's a leftover from your previous implementation.